test/integration/bats/custom-share-folders.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+
+load ${BASE_TEST_DIR}/helpers.bash
+
+use_disposable_machine
+
+@test "$DRIVER: create with custom shared folders" {
+  run machine create -d $DRIVER --parallels-share-folder ~/Documents --parallels-share-folder /Library/Application\ Support $NAME
+}
+
+@test "$DRIVER: shared folder with a tilda (~) is mounted properly" {
+  run machine ssh $NAME -- "mount | grep prl_fs | awk -F ' on | type ' '{ print \$2 }'"
```

Comment:
Just to inform: to list mounts of a specific FS type one can use `mount -t $FS`.

Reply:
Thanks! I will use `mount -t` instead 👍